lib/Sema/TypeCheckAccess.cpp

```diff
@@ -2040,7 +2040,7 @@ swift::getDisallowedOriginKind(const Decl *decl,
       where.getDeclContext()->getAsDecl()->getModuleContext()->isResilient() &&
```

Comment:
Just to clarify: `getModuleContext() ~> __ObjC`, whereas `getModuleContextForNameLookup() ~> theRealClangModule`?

If so, why isn't `getModuleContextForNameLookup()` required here? Or is it the case that `__ObjC` is never considered resilient?